src/Microsoft.Data.SqlClient/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -6,11 +6,14 @@ internal static partial class Interop
 {
     internal static partial class Libraries
     {
-        internal const string Crypt32 = "crypt32.dll";
         internal const string Kernel32 = "kernel32.dll";
         internal const string NtDll = "ntdll.dll";
 #if !NET8_0_OR_GREATER
+        internal const string Crypt32 = "crypt32.dll";
         internal const string SspiCli = "sspicli.dll";
+#endif
```

Comment:
Can we consolidate this into the `#if NETFRAMEWORK` below? I think it reads better given we've dropped .NET 6 and 7.

Reply:
Don't worry too much, this file will be dropped shortly.

Reply:
Having trouble keeping track 😆